woodwork/table_schema.py

```diff
@@ -552,7 +552,12 @@ def _filter_cols(self, include=None, exclude=None, col_names=False):
 
         return cols_to_return
 
-    def _get_subset_schema(self, subset_cols):
+    @property
```

Comment:
I think if you change this to a property like this, EvalML will break since they will lose the ability to pass in `subset_cols`, no?

Reply:
The property serves as an alias that returns the actual function, so it wouldn't cause any breaking changes.

```python
class A:
    def function(self, x):
        return x
    
    @property
    def aliase(self):
        return self.function

assert A().function(True) is True
```

Reply:
Yeah, when I first looked, I thought you were calling the other method instead of returning it. Ignore my original comment.